conf/postgres-operator/pgpool-template.json

```diff
@@ -44,7 +44,7 @@
 		    {{.ContainerResources }}
                     "env": [{
                         "name": "PG_USERNAME",
-                        "value": "testuser"
+                        "value": "{{.PGUser}}"
```

Comment:
Is .PGUser correct here? it is the only place I see it, where I see "User" in several other places.

Reply:
No its not. Ill fix it